### ISSUE TITLE
Investigate - what happens to unpublished content on Sections

### ISSUE THREAD

**Issue by Bellfalasch** [enonic/cms2xp#27]
What happens to their data? Stored as "offline" in XP with `publish.from` and `publish.first`?

**Comment by Bellfalasch** [comment]
Could this be researched? In CMS we will have MenuItems of type "Section", these will have multiple content published to them. Sometimes these are unpublished, meaning they're still kept on the menuitem but not shown outside of admin. How does CMS2XP handle these?

**Comment by sigdestad** [comment]
Only items in sections that are "published/active" should be included in the section property. The rest we can safely drop

**Comment by alansemenov** [comment]
@Bellfalasch is it a question? do you still need an answer/additional action?

**Comment by Bellfalasch** [comment]
Yes, asking for confirmation. This was to my knowledge never done and I believe cms2xp doesn't unpublish things that was unpublished this way in CMS.
